Xamarin.Essentials/Calendars/Calendars.uwp.cs

```diff
@@ -322,7 +322,7 @@ static async Task<bool> PlatformUpdateCalendarEvent(CalendarEvent eventToUpdate)
             var existingEvent = await GetEventByIdAsync(eventToUpdate.Id);
 
             Appointment thisEvent = null;
-            var instance = await CalendarRequest.GetInstanceAsync();
+            var instance = await CalendarRequest.GetInstanceAsync(AppointmentStoreAccessType.AllCalendarsReadWrite);
```

Comment:
Is this failing because we need to specify a different permission in manifest file - there was a second calendar permission that allowed for more access I seem to recall

Reply:
that rescap thing that you mentioned a while back I could never get working, uwp doesn't seem to understand how to handle it. This was an attempt to see if I could get enough permissions to access calendar events outside of xamarin created calendars, but not sure if it worked off the top of my head, will probably remove at this stage.